src/graphql/queries/patients.ts

```diff
@@ -22,7 +23,7 @@ export interface Patient {
   gender: string;
   age: number;
   runNumber: number;
-  barcodeValue: number;
+  barcodeValue: string;
```

Comment:
@aaronabraham311 this may affect your barcode-related changes (also for patient mutations)